runway/hooks/staticsite/auth_at_edge/templates/refresh_auth/__init__.py

```diff
@@ -0,0 +1,147 @@
+# pylint: skip-file
+"""Refresh the authorization token for new credentials.
+
+NOTE: pylint: skip-file has been added to this file to avoid an issue with a false
+positive about duplicated code. Please remove during iteration.
+"""
+
+import logging
+import traceback
+from urllib.parse import parse_qs
+
+from shared import (create_error_html, extract_and_parse_cookies, get_config,  # noqa pylint: disable=import-error
+                    get_cookie_headers, http_post_with_retry)
+
+LOGGER = logging.getLogger(__name__)
+
+
+def handler(event, _context):
+    """Handle the authorization refresh.
+
+    Keyword Args:
+        event: The Lambda Event
+    """
+    request = event['Records'][0]['cf']['request']
+    config = get_config()
```

Comment:
Same comment as above -- get_config() should be able to be run outside the handler